receiver/otlpreceiver/config.md

```diff
@@ -0,0 +1,109 @@
+# Otlpreceiver Configuration Reference
+
+This is an autogenerated file that can be used as a reference to create a collector configuration section for this component.
+It contains all of the configuration fields and sub-fields accepted by this component. The fields are grouped by 'level' and
+parent field, with the top level fields at the top. For compound fields, navigate to lower level fields by clicking on any
+highlighted link in the Type column.
+
+Columns:
+
+* **Name**: The name of the field. This is the field's map key in the yaml config.
+* **Type**: The golang type that the field will be converted to. Clickable to subsequent points in the document for compound
+fields.
+* **Default**: The default value of the field if no value is specified.
+* **Docs**: Extracted inline comments explaining field usage, acceptable values, etc.
+## otlpreceiver-Config
```

Comment:
Can we drop the `otlpreceiver-` prefix?

Reply:
👍

Reply:
Yes, this is done, but only for the first markdown table. Is this what you wanted? The other tables need to have unique names since they're being used as anchor targets.